### DISCUSSION TITLE
如何在首页展示最近新建/更改的 page？

### DISCUSSION THREAD

**Discussion by AugustHuangMX** [oldwinter/knowledge-garden#70] [提问]
是使用 ob 的 dataview 吗？

**Comment by PinkR1ver** [comment]
same question +1

**Comment by PinkR1ver** [comment]
> 是使用 ob 的 dataview 吗？

我使用的是最基础的quartz+obsidian的方案，在obsidian里使用dataview可以看到，但是在web端没有渲染出来

**Reply by PinkR1ver** [reply]
https://github.com/jackyzha0/quartz/issues/102
原生quartz暂时还没有渲染dataview的功能

**Reply by PinkR1ver** [reply]
https://garden.oldwinter.top/LATEST/
原作者的一些界面似乎也不能渲染dataview
